README.md

```diff
@@ -310,6 +316,19 @@ You can exclude pages from the build by either moving them out of `src/pages` or
 
 ### Links
 
+#### Anchor links for headings
+
+By default, the theme automatically generates an anchor link for headings h2 – h6, using the kabab-case (lowercase words separated by hyphens).
+For example, if the heading name is **Inline code example**, the theme creates the following anchor link: `#inline-code-example`.
+
+You can customize the anchor link for a heading using the following syntax:
+
+```markdown
+## Inline code example {#my-custom-anchor-link}
+```
+
+This will override the auto-link behavior and create a custom anchor link.
```

Comment:
Mention or illustrate that you don't have to maintain kabab-case in the anchor link.

Reply:
Good call. Done.